0074

```diff
@@ -0,0 +1,164 @@
+
+Current Hotspot self-reported parameters like Heartbeats and Speed Tests are limited in providing information about the quality of coverage. Neither takes into account the directionality of Radios, environmental obstructions that prevent the propagation of signals, and while they are hard to spoof, they are not verified by external sources.
+
+Once implemented, Modeled Coverage will be a significant milestone in introducing the Proof-of-Coverage concept in the Mobile Network. It's covered in a detailed MOBILE PoC roadmap in the blog post MOBILE Proof-of-Coverage: The Road Ahead [MOBILE Proof-of-Coverage: The Road Ahead](<https://blog.helium.com/mobile-proof-of-coverage-the-road-ahead-73a25601a13d>) as Obstruction Data.
+Combined with other parameters like uptime (Heartbeats) and backhaul (Speed Test), it allows for more fair rewards and incentivizes the deployment of Radios at optimal locations.
+
+
+# Motivation
+
+Building the MOBILE Network began with the Genesis Period, during which 5G Hotspot owners with Radios had to send just one Heartbeat in 24 hours to prove they were online. This was a necessary starting point to kickstart the deployments. However, from the start, the goal was to introduce more and more data sources to evaluate the certainty and usefulness of coverage.
+
+To be successful, any Network must be reliable, always available, and meet the expectations of its users. To move one step closer to the Proof-of-Coverage verified by external sources, we propose to use Modeled Coverage data to evaluate performance of Radios.
+
+
+# Stakeholders
+
+This HIP affects only MOBILE SubDAO, particularly 5G Hotspot owners with Radios and users of the MOBILE Network.
+
+5G Hotspot owners: 5G Hotspot owners with Outdoor Radios will need to meet new quality coverage requirements to earn maximum MOBILE rewards. Some owners might need to adjust the locations and/or angles of their Outdoor Radios to optimize for coverage. It is possible that in some places, it won't be possible to earn max rewards due to environmental obstacles that are not possible to mitigate. This HIP also proposes to consider rewarding those deployers who were among the first to provide quality coverage in a given Hex.
+
+There will be three possible outcomes for owners of Radios:
+<ol>
+<li>Increase in earnings. Because some Radios on the MOBILE Network won't be able to meet minimum quality requirements, part of the rewards will be re-distributed proportionally to those who qualify based on all PoC criteria.</li>
+<li>Earnings will stay the same.</li>
+<li>Less or no earnings.</li>
+</ol>
+
```

Comment:
Does this impact manufacturers in any way? Do third-party mobile hotspot apps have to add anything to be compliant? Or get customers the information they need?

Reply:
No, proposed changes do not impact manufacturers, and no action is needed on the third-party mobile apps.